sys/Makefile.dep

```diff
@@ -46,4 +46,15 @@ ifneq (,$(filter shell_commands,$(USEMODULE)))
   endif
 endif
 
+# for convenience, we base the inclusion of the dbgpin module on the value of
+# the DBGPIN_PINS env variable
+ifneq (,$(DBGPIN_PINS))
+  FEATURES_REQUIRED += periph_gpio
+  USEMODULE += dbgpin
+  # XXX native segfaults when not using the mock module
+  ifeq (native,$(BOARD))
+    USEMODULE += periph_gpio_mock
+  endif
```

Comment:
@benpicco any idea why this might be needed?

Reply:
Without this the real GPIOs (e.g. `/dev/gpiochip0`) are used.
If there are none, all calls to `gpio_init()` should fail.

Reply:
weirdly, only the calls to `set/clear` trigger a segfault... But this makes sense. What do you think would be the most sensible way to proceed here?